backend/audit/models.py

```diff
@@ -206,22 +208,36 @@ class STATUS:
 
     def validate_full(self):
         """
+        Full validation, intended for use when the user indicates that the
+        submission is finished.
+
+        Currently a stub, but eventually will call each of the individual
+        section validation routines and then validate_cross.
+        """
+
+        validation_methods = []
+        errors = [f(self) for f in validation_methods]
+
+        if errors:
+            return errors
+
+        return self.validate_cross()
+
+    def validate_cross(self):
+        """
+        This method should NOT be run as part of full_clean(), because we want
+        to be able to save in-progress submissions.
+
         A stub method to represent the cross-sheet, “full” validation that we
         do once all the individual sections are complete and valid in
         themselves.
         """
-        all_sections = [
-            self.general_information,
-            self.federal_awards,
-            self.corrective_action_plan,
-            self.findings_text,
-            self.findings_uniform_guidance,
-            self.additional_ueis,
-            self.notes_to_sefa,
-        ]
-        if all(section for section in all_sections):
-            return True
-        return True
+        shaped_sac = audit.cross_validation.sac_validation_shape(self)
+        all_functions = audit.cross_validation.functions
```

Comment:
**comment (non-blocking)**: In context, the general term `functions` doesn't seem like it describes what this list is very well. Perhaps `all_validations` or `validation_functions`?